weaviate/gql/get.py

```diff
@@ -1002,6 +1005,13 @@ def with_alias(
         self._alias = alias
         return self
 
+    def with_consistency_level(self, consistency_level: ConsistencyLevel):
+        """Set the consistency level for the request."""
+
```

Comment:
Should we check if the `consistency_level` is really of `ConsistencyLevel` data type?

Reply:
@parkerduckworth ?

Reply:
we can add it later if someone complains